api/envoy/extensions/load_balancing_policies/common/v3/common.proto

```diff
@@ -42,8 +43,22 @@ message LocalityLbConfig {
     // failing service.
     bool fail_traffic_on_panic = 3;
 
-    // If set to true, Envoy will force LocalityDirect routing if a local locality exists.
-    bool force_locality_direct_routing = 4;
+    // If set to true, Envoy will always route traffic to the local zone regardless of the
+    // upstream zone structure. In Envoy's default configuration, traffic is distributed proportionally
+    // across all upstream hosts while trying to maximize local routing when possible. The approach
+    // with force_local_zone aims to be more predictable and if there are upstream hosts in the local
+    // zone, they will receive all traffic.
+    // * :ref:`runtime values <config_cluster_manager_cluster_runtime_zone_routing>`.
+    // * :ref:`Zone aware routing support <arch_overview_load_balancing_zone_aware_routing>`.
+    bool force_local_zone = 4;
```

Comment:
This is a breaking API change, I think we should create a new field here and deprecate the old one if it's no longer applicable.

Reply:
Sure, will update that now. FWIW the `force_locality_direct_routing` was only merged in yesterday and hasn't made it into an official release yet.

Reply:
Oh if it was just merged yesterday then leave it as is.  I thought the field existed for a while.

Reply:
All good, thanks!